changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1 | 2020-03-23
+* Drop matplotlib < 3 requirement.
+* Add functionality which redirects users to API when accessing particular fields.
```

Comment:
Not sure this is going to clear to readers. An example would be nice: accessing `file["FD curves"]` will throw an error and redirect users to use `file.fd_curves`.

Reply:
Changed the line to: 
* Add functionality which redirects users to API when accessing particular fields, e.g. accessing `file["FD curve"]` will throw an error and redirect users to use `file.fdcurves`.